docs/cmake-modules.rst

```diff
@@ -21,6 +22,7 @@ They can be included using ``find_package``:
     find_package(Cython REQUIRED)
     find_package(NumPy REQUIRED)
     find_package(PythonExtensions REQUIRED)
+    find_package(F2PY REQUIRED)
```

Comment:
```
find_package(F2py REQUIRED)
```

Reply:
Not the right fix. I renamed the module.